src/Adapter/Cart/CartPresenter.php

```diff
@@ -261,7 +261,7 @@ public function addCustomizedData(array $products, Cart $cart)
     }
 
     /**
-     * @param $cart
+     * @param Cart $cart
```

Comment:
Could be cool to remove the `!is_a($cart, 'Cart')` condition below and `Cart $cart` in signature :) wdyt @mickaelandrieu @tomlev ?
(Was added in 769f22b9d6b475f198effabe68d5810839cc7939)

Reply:
Good idea! But as this PR haven't changed since your comment, we will have to create another PR for this.